docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css

```diff
@@ -225,3 +225,13 @@ body.gin--edit-form .views-exposed-form .select-preact button svg {
 .gin--edit-form .field--widget-paragraphs .field-multiple-table {
   display: table;
 }
+
+/* Styles to Roles */
```

Comment:
Can you please change this comment to something more descriptive, like "Role description styles"?